### ISSUE TITLE
Instances used for training

### ISSUE THREAD

**Issue by alpguvenir** [KSH0660/BrainTumor#2]
In the Brats 2018 dataset there are:
- 210 instances under HGG folder
- 75 instances under LGG folder

In the paper it states that among these 285 instances 28 were used for validation loss and 190 were used for training purposes, is there any information on which of the instances were used among the whole dataset?

**Comment by KSH0660** [comment]
Hello @alpguvenir, sorry for late reply.
I'm sorry I couldn't present it clearly now. We selected 65 subjects (have low quality, e.g., motion blurring, ..) subjectively without any clear criteria. But it is quite simple task (just took 1 hour). We recommend you to build clear criteria or just use our uploaded network weight please.
This selection procedure was a quite important step for in-painting quality. The low quality images should have been removed in training dataset.
